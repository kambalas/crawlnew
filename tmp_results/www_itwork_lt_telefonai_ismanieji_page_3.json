[
  {
    "name": "Apple iPhone 16 Pro Max | Desert Titanium | 6.9 | Super Retina XDR display with ProMotion | 2868 x 1320 pixels | A18 Pro | Internal RAM 8 GB | 1000 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "2,141.13€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max | Black Titanium | 6.9 | Super Retina XDR | 2868 x 1320 pixels | Apple | A17 Pro | Internal RAM 8 GB | 256 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,548.32€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max | Black Titanium | 6.9 | Super Retina XDR display with ProMotion | 2868 x 1320 pixels | A18 Pro | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,826.03€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 512GB White Titanium | Apple",
    "price": "1,883.14€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 1TB Natural Titanium | Apple",
    "price": "1,949.38€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 1TB Natural Titanium",
    "price": "1,926.50€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 1TB Desert Titanium | Apple",
    "price": "1,949.38€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 1TB Black Titanium | Apple",
    "price": "1,949.38€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 16 cm (6.3) Dual SIM iOS 18 5G USB Type-C 256 GB Titanium",
    "price": "1,383.62€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | White | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,568.65€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | White | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,568.65",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | White | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 256 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,336.22",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | White | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 128 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,168.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Ultramarine | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 256 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,313.80",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Ultramarine | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 128 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,168.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Ultramarine | 6.7 | Super Retina XDR display | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,575.63",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Teal | 6.7 | Super Retina XDR | 2796 x 1920 pixels | A18 | Internal RAM 8 GB | 128 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,168.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Pink | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 128 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,168.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Pink | 6.7 | Super Retina XDR display | 2796 x 1290 pixels | Apple | A18 | Internal RAM 8 GB | 256 GB | Dual SIM | 4G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,336.22",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Pink | 6.7 | Super Retina XDR display | 2796 x 1290 pixels | Apple | A18 | Internal RAM 8 GB | 256 GB | Dual SIM | 4G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,336.22€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Pink | 6.7 | Super Retina XDR display | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,568.65€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Black | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,575.63€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Black | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 256 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,313.80€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Black | 6.7 | 2796 x 1290 pixels | Internal RAM 8 GB | 128 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1,168.75€",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB Black",
    "price": "1,092.85€",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 256GB - Natural Titanium",
    "price": "1,453.72€",
    "error": false
  },
  {
    "name": "Apple iPhone 15 15.5 cm (6.1) Dual SIM iOS 17 5G USB Type-C 128 GB Blue",
    "price": "854.19€",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 17 cm (6.7) Dual SIM iOS 16 5G 256 GB Red",
    "price": "967.19€",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 17 cm (6.7) Dual SIM iOS 16 5G 256 GB Purple",
    "price": "981.12€",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 17 cm (6.7) Dual SIM iOS 16 5G 256 GB Black",
    "price": "981.12€",
    "error": false
  },
  {
    "name": "Apple iPhone 14 15.5 cm (6.1) Dual SIM iOS 16 5G 512 GB White",
    "price": "1,009.15€",
    "error": false
  },
  {
    "name": "Apple iPhone 14 128GB Midnight",
    "price": "731.86€",
    "error": false
  },
  {
    "name": "Apple iPhone 13 128GB - Starlight",
    "price": "623.34€",
    "error": false
  }
]