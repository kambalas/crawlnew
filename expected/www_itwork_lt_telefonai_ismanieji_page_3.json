[
  {
    "name": "Apple iPhone 16 Plus | Black | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 256 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1313.80",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | Black | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 128 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1168.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB Black",
    "price": "1095.04",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 256GB - Natural Titanium",
    "price": "1456.62",
    "error": false
  },
  {
    "name": "Apple iPhone 15 15.5 cm (6.1) Dual SIM iOS 17 5G USB Type-C 128 GB Blue",
    "price": "855.91",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 17 cm (6.7) Dual SIM iOS 16 5G 256 GB Red",
    "price": "966.84",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 17 cm (6.7) Dual SIM iOS 16 5G 256 GB Purple",
    "price": "980.77",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 17 cm (6.7) Dual SIM iOS 16 5G 256 GB Black",
    "price": "980.77",
    "error": false
  },
  {
    "name": "Apple iPhone 14 15.5 cm (6.1) Dual SIM iOS 16 5G 512 GB White",
    "price": "1008.80",
    "error": false
  },
  {
    "name": "Apple iPhone 13 128GB - Starlight",
    "price": "624.60",
    "error": false
  },
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "name": "Apple iPhone 16 Pro Max | Black Titanium | 6.9 | Super Retina XDR display with ProMotion | 2868 x 1320 pixels | A18 Pro | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1554.48",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 512GB White Titanium | Apple",
    "price": "1554.48",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 1TB White Titanium | Apple",
    "price": "1611.05",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 1TB Natural Titanium | Apple",
    "price": "1611.05",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 1TB Natural Titanium | Apple",
    "price": "1611.05",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 1TB Desert Titanium | Apple",
    "price": "1611.05",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 1TB Black Titanium | Apple",
    "price": "1660.96",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 16 cm (6.3) Dual SIM iOS 18 5G USB Type-C 256 GB Titanium",
    "price": "1143.49",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | White | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 512 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1296.40",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus | White | 6.7 | Super Retina XDR | 2796 x 1290 pixels | A18 | Internal RAM 8 GB | 256 GB | Dual SIM | 4G | 5G | Main camera resolution 48+12 MP | Secondary camera resolution 12 MP | iOS | 18",
    "price": "1168.75",
    "error": false
  },
  {
    "index": 2,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  }
]